openadapt/db/crud.py

```diff
@@ -391,7 +403,9 @@ def get_action_events(
         list[ActionEvent]: A list of action events for the recording.
     """
     assert recording, "Invalid recording."
-    action_events = _get(session, ActionEvent, recording.id)
+    action_events = _get(
+        session, ActionEvent, recording.id, eager=True, relationships=["screenshot"]
```

Comment:
Why not `relationships=["recording", "screenshot", "window"]`?

Reply:
can add `recording` but not window